cli/.eslintignore

```diff
@@ -1,14 +1,21 @@
 **/__snapshots__
 /build
 
-# cli/types is linted by tslint/dtslint
-/types
-
 # these are all copied from dist'd builds from the individual libs
 /angular
 /react
 /react18
 /vue
 /vue2
 /svelte
-/mount-utils
\ No newline at end of file
+/mount-utils
+
+/types/bluebird/**/*
```

Comment:
We only lint and format our own types, not the ones we copy-paste from node_modules.